src/libs/actions/PersonalDetails.js

```diff
@@ -323,7 +310,6 @@ function fetchLocalCurrency() {
         .then((data) => {
             currency = data.currency;
         })
-        .then(getCurrencyList)
         .then(() => {
             Onyx.merge(ONYXKEYS.MY_PERSONAL_DETAILS, {localCurrencyCode: currency});
```

Comment:
@luacmartins What I mentioned [here](https://github.com/Expensify/Web-Expensify/pull/34151#discussion_r919249607), this is how we currently store and merge the data about `localCurrencyCode`

Reply:
Yea, I'm planning on changing this PR based on the changes [here](https://github.com/Expensify/App/pull/9560)